conftest.py

```diff
@@ -1,13 +1,25 @@
+"""Pytest configuration and fixtures for virtualizarr tests."""
```

Comment:
I reorganized this file as it includes fixtures, variables and helper functions of various purpose.